docs/application/dotnet/guides/multimedia/camera.md

```diff
@@ -210,11 +209,11 @@ To configure the camera:
         camera.CaptureCompleted += CaptureCompletedEventHandler;
         ```
 
-## Setting the Display for the Camera Preview
+## Setting the display for the camera preview
```

Comment:
Gerund forms should be avoided for headings.
Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#verb-form
```suggestion
## Set the display for the camera preview
```